src/modules/zfshostid/zfshostid.schema.yaml

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2022 Anke Boersma <demm@kaosx.us>
+# SPDX-License-Identifier: GPL-3.0-or-later
+---
+$schema: https://json-schema.org/schema#
+$id: https://calamares.io/schemas/zfshostid
+additionalProperties: false
+type: object
+
+required:
+    - zfs
```

Comment:
I think this means that the item "zfs" is required.  Not the module zfs.  I am not sure that you need a schema for a module which has no config.

I think module requirements are handled by setting `requiredModules` in module descriptor.